test/LoadTest/cache_refresh/README.md

```diff
@@ -0,0 +1,131 @@
+# Cache refresh Tests
+
+These tests are used to verify the cache refresh in a balanced system.
+
+
+#### Requirements
+
+- Bash shell (linux)
+- Jmeter (2.11 verson or higher)
+- Python (2.7.x version)
+- Nginx (1.0.15 or higher)
+
+
+#### Architecture for testing
+
+- 1 node to balancer (nginx)
+- 2 nodes to context broker installed
+- 1 node to mongo db (associated to CBs)
+- 1 node to launch listener and viewer
+- 1 node to launch jmeter
+
+
+#### Scripts used:
+
+- **append_entities_long_time_v1.0.jmx**: Jmeter script used to generate N Append Requests with an estable TPS. Additionally is possible to create a subscription before all append requests.
+     
+  Parameters:
+  ```
+     HOST: CB or balancer host (by default, localhost)
+     PORT: CB or balancer port (by default, 1026)
+     THREADS: number of threads used in the test (by default, 1)
+     TESTTIME: duration time of the test (by default, 60)
+     SERVICE: service header used in requests (by default, entities_append_long_time)
+     SERVICE_PATH: service path header used in requests (by default, /test)
+     ENTITIES: number of consecutive entities in each thread (by default, 10)
+     SUBSCRIPTION: optionally is possible to create a subs before all appends requests, in case of would like to create use CREATE as value (by default, nothing)
```

Comment:
a subs -> a sub

or

a subs -> some subs

"in case of would like" -> "in case you would like" (?)